stable/aws-sigv4-proxy-admission-controller/templates/priorityclass.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.priorityClass.create -}}
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: {{ template "aws-sigv4-proxy-admission-controller.priorityClassName" . }}
+  labels:
+{{ include "aws-sigv4-proxy-admission-controller.labels" . | indent 4 }}
+value: 1000000
+preemptionPolicy: PreemptLowerPriority
+globalDefault: false
+description: "This priority class is with PreemptLowerPriority for AWS sigv4 proxy."
+{{- end -}}
```

Comment:
Any reason why you wouldn't want to use the built-in `system-cluster-critical` priority class?